README.md

```diff
@@ -71,19 +71,13 @@ For more information please see `./CONTRIBUTING.md`.
 
 This library should always compile with any combination of features on **Rust 1.48.0**.
 
-To build with the MSRV you will need to pin `serde` (if you have the feature enabled)
+To build with the MSRV, you will need to use the Cargo-minimal.lock file.  **Note:** the MSRV will not get the latest dependency updates.
 
 ```
-# serde 1.0.157 uses syn 2.0 which requires edition 2021
-cargo update -p serde_json --precise 1.0.99
-cargo update -p serde --precise 1.0.156
-cargo update -p quote --precise 1.0.30
-cargo update -p proc-macro2 --precise 1.0.63
-cargo update -p serde_test --precise 1.0.175
+cp Cargo-minimal.lock Cargo.lock
+cargo build
```

Comment:
In d2325dbb36de28aecf883c7cf6f0a044329f5666:

This reverts https://github.com/rust-bitcoin/rust-bitcoin/pull/1982 which we just merged 3 weeks ago. It does not accomplish the same thing.

Reply:
This is just a copy paste of #2074 

I will remove this commit once something is merged to master to fix the pinning issue.

Reply:
#2074 is wrong.

Reply:
> This is just a copy paste of https://github.com/rust-bitcoin/rust-bitcoin/pull/2074

It would be better to rebase your branch off of the change in the future so people don't get confused and think it's part of this PR.

Reply:
It would still show up in the diff.

Reply:
> It would still show up in the diff.

Yes, that's true, but if the commit is examined it will show a different author.